fetch/api/request/request-keepalive-quota.html

```diff
@@ -0,0 +1,93 @@
+﻿<!doctype html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <title>Request Keepalive Quota Tests</title>
+        <meta name="help" href="https://fetch.spec.whatwg.org/#request">
+        <meta name="help" href="https://fetch.spec.whatwg.org/#body-mixin">
+        <meta name="author" title="Microsoft Edge" href="https://www.microsoft.com">
+        <meta name="timeout" content="long">
+        <script src="/resources/testharness.js"></script>
+        <script src="/resources/testharnessreport.js"></script>
+    </head>
+    <body>
+        <script>
+            "use strict";
+
+            // We want to ensure that our keepalive requests hang slightly before completing so we can validate
+            // the effects of a rolling quota. To do this we will utilize trickle.py with a 1s delay. This should
+            // prevent any of the Fetch's from finishing in this window.
+            var trickleURL = "../resources/trickle.py?count=1&ms=";
+            var standardDelay = 1000;
+
+            // We should expect 64KB of rolling quota for any type of keep-alive request sent.
```

Comment:
Yeah, validated using Edge's implementation. Rolling keepalive quotas will be available for fetch w/keepalive and sendBeacon in an upcoming insider build.